docs/source/admin-guide/install.rst

```diff
@@ -2,31 +2,139 @@
-the Numba_ channel on `Anaconda Cloud <https://anaconda.org/numba>`_::
+the Numba_ development channel on `Anaconda Cloud <https://anaconda.org/numba>`_::
 
-    conda install --channel numba llvmlite
+    conda install -c numba/label/dev llvmlite
 
 Binary wheels are also available for installation from PyPI_::
 
     pip install llvmlite
 
+Development releases for binary wheels are not made available.
+
+Contrary to what you might expect, the llvmlite packages built by the Numba
+maintainers do *not* use any LLVM shared libraries that may be present on the
+system, and/or in the Conda environment. The parts of LLVM required by llvmlite
+are statically linked at build time.  As a result, installing llvmlite from a
+binary package does not also require the end user to install LLVM.  (For more
+details on the reasoning behind this, see: :ref:`faq_why_static`). Note however
+also that llvmlite packages compiled by other parties, e.g. conda-forge may
+split this into and ``llvmlite`` and ``llvm`` package.
+
+Conda packages:
+---------------
+
+The Numba maintainers ship to the Numba channel:
+
+  * Numba packages
+  * llvmlite packages
+  * llvmdev packages (this contains a build of LLVM)
+
+The llvmdev packages are not needed at runtime by llvmlite packages as
+llvmlite's dynamic libraries are statically linked (see above) at compile time
+against LLVM through the dependency on the ``llvmdev`` package.
+
+The Anaconda distribution and conda-forge channels ship:
+
+  * Numba packages
+  * llvmlite packages
+  * LLVM split into runtime libraries (package called ``llvm``) and compile time
+    libraries/headers etc this contains a build of LLVM (package called
+    ``llvmdev``)
+
+At compile time the ``llvmdev`` and ``llvm`` packages are used to build llvmlite and
+llvmlite's dynamic libraries are dynamically linked against the libraries in the
+``llvm`` meta-package. This means at runtime ``llvmlite`` depends on the ``llvm``
+package which has the LLVM shared libraries in it (it's actually a package
+called ``libllvm`` that contains the DSOs, but the ``llvm`` package is referred to
+so as to get the ``run_exports``).
+
+Using `pip`
+-----------
+
+The Numba maintainers ship binary wheels:
+
+  * Numba wheels (``x86*`` architectures)
+  * llvmlite wheels (``x86*`` architectures)
+
+Note that the llvmlite wheels are statically linked against LLVM, as per the
+conda packages on the Numba channel. This mitigates the need for a LLVM based
+binary wheel. Not also that this, as of version 0.36 does not include the
+``aarch64`` architectures, for example installation on a RaspberryPis is not
+supported.
+
+The Numba maintainers ship an ``sdist`` for:
+
+  * Numba
+  * llvmlite
+
+Note that there is no ``sdist`` provided for LLVM. If you try and build ``llvmlite``
+from ``sdist`` you will need to bootstrap the package with your own appropriate
+LLVM.
+
+How this ends up being a problem.
```

Comment:
Should this be a sub-title?

Reply:
probably, but then the later `Things to "fix" it...` should be one also, perhaps?

Reply:
done